src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -25,6 +25,10 @@ def fetch(self, sql) -> Iterator[any]:
     def save_table(self, full_name: str, rows: list[any], mode: str = "append"):
         raise NotImplementedError
 
+    @abstractmethod
```

Comment:
```suggestion
```

Doesn't need to be abstract, implement it here

Reply:
Done